styles/styles.css

```diff
@@ -145,45 +155,51 @@ a:hover {
 a.button:any-link,
 button {
   font-family: var(--body-font-family);
-  display: inline-block;
   box-sizing: border-box;
   text-decoration: none;
-  border: 2px solid transparent;
-  padding: 5px 30px;
-  text-align: center;
   font-style: normal;
-  font-weight: 600;
   cursor: pointer;
-  color: var(--background-color);
-  background-color: var(--link-color);
-  margin: 16px 0;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
-  border-radius: 30px;
+  margin: 0;
+  padding: 0 30px;
+  vertical-align: middle;
+  font-size: 0.875rem;
+  line-height: 38px;
+  text-align: center;
+  display: inline-block;
+  transition: 0.1s ease-in-out;
+  transition-property: color, background-color, background-position,
+    background-size, border-color, box-shadow;
+  font-weight: 600;
+  text-transform: uppercase;
+  border-radius: 0;
+  background-color: var(--button-color);
+  color: #fff;
+  border: 1px solid var(--button-color);
+}
+
+a.button:active {
+  background-color: #c00;
+  color: #fff;
```

Comment:
This is not required as it's same as line 179

Reply:
Removed